util/ziputil/zip_writer.go

```diff
@@ -26,6 +27,27 @@ func WriteZipFromFiles(w io.Writer, files []string, compress bool) error {
 		}
 	}
 
+	if withREADME {
```

Comment:
This place doesn't sounds to be right, as it is a "zip utility", and should have no logics about "cpu" or "profiling" or "readme".